app/assets/javascripts/teams.js

```diff
@@ -1,5 +1,5 @@
 jQuery(function ($) {
-  $('#add_team_user_btn').on('click', function (_event) {
+  $('#add_team_user_btn').on('click', function () {
```

Comment:
Yes, it's optional. By default there's no check on how functions are referenced/called. JS sucks on those kind of stuff and that's one of the reasons why TypeScript is getting more and more popular and being used in bigger projects.

Just saw that you've configured ESLint to ignore unused variables starting with `_`. I guess you can remove that and let ESLint do the job of checking. 

I know that eventually there will be a case like:

```js
(not_used, used) => {
  // ...
}
```

and you can use `_not_used` without needing to configure ESLint to not complain about it. ESLint is smart enough to know that `used` is used and the first parameter is necessary to be declared even if not used. So tou don't need to configure to ignore variables starting with `_`.

On the other hand, keeping the `_var` indicates that the function receives that parameter but it's not being used in it. If an app gets too big and lacks documentation, having that artifact would help a lot. In this case, it's a well documented and know scenario of listening to a dom event through jQuery. I don't think it's necessary, but it's your call to keep it or not.

I also realized that `event` is a reserved keyword and that's another reason why ESLint didn't catch the erro of `event` not being defined. For events I'm comfortable with `e` or `evt`.

What do you think?